mvc/views/view-components/sample/TodoList/src/TodoList/ViewComponents/PriorityListViewComponent.cs

```diff
@@ -0,0 +1,29 @@
+﻿//#define First
+#if First
+
+using System.Linq;
+using Microsoft.AspNet.Mvc;
+using TodoList.Models;
+
+namespace TodoList.ViewComponents
```

Comment:
Why bother showing both of these? There's not really a substantial difference between sync and async. Just the method name changes.